src/middlewared/middlewared/etc_files/local/smb4.conf

```diff
@@ -148,7 +148,12 @@
                 pc.update({'server min protocol': 'SMB2_02'})
                 pc.update({'unix extensions': 'No'})
             if db['cifs']['guest'] != "nobody":
-                pc.update({'guest account': db['cifs']['guest']})
+                try:
+                    middleware.call_sync("user.get_user_obj", {"username": db['cifs']['guest']})
+                    pc.update({'guest account': db['cifs']['guest']})
+                except KeyError:
+                    logger.warning("SMB guest user [%s] does not exist. Defaulting to [nobody].", db['cifs']['guest'])
```

Comment:
Where does it get defaulted to `nobody`?

Reply:
https://www.samba.org/samba/docs/current/man-html/smb.conf.5.html#GUESTACCOUNT